Src/BootCamp.Chapter/CredentialsManager.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.IO;
+using System.Text;
+
+namespace BootCamp.Chapter
+{
+    internal class CredentialsManager
+    {
+        public struct Person
+        {
+            public Person(string name, string password)
+            {
+                Name = name;
+                Password = password;
+            }
+
+            public string Name { get; set; }
+            public string Password { get; set; }
+        }
+
+        internal static void Demo()
+        {
+            Console.WriteLine("Would you like to register or login");
+            Console.WriteLine("Choose R voor register");
+            Console.WriteLine("Choose L voor login");
+            var choice = Console.ReadLine();
+            switch (choice)
+            {
+                case "R":
+                    Register();
+                    break;
+
+                case "L":
+                    Login();
+                    break;
+
+                default:
+                    Console.WriteLine("This is not a right choice. Only R and L are valid");
+                    break;
+            }
+        }
+
+        private static void Login()
+        {
+            var isValidLogin = false;
+            var data = File.ReadAllText(@"Database/user.txt");
```

Comment:
How is data different from input?

Reply:
data for me is the data from the file and input for me is the input from the user

Reply:
or is that also wrong naming